roles/bootstrap-os/tasks/main.yml

```diff
@@ -25,6 +25,9 @@
 - include_tasks: bootstrap-opensuse.yml
   when: '"openSUSE" in os_release.stdout'
 
+- include_tasks: bootstrap-oracle.yml
+  when: '"Oracle" in os_release.stdout'
```

Comment:
Should this be `OracleLinux` ?

Reply:
I don't think any other distros will put "Oracle" in their os-release file willingly...

Reply:
```
[vagrant@oracle7 ~]$ cat /etc/os-release
NAME="Oracle Linux Server"
VERSION="7.5"
ID="ol"
VERSION_ID="7.5"
PRETTY_NAME="Oracle Linux Server 7.5"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:oracle:linux:7:5:server"
HOME_URL="https://linux.oracle.com/"
BUG_REPORT_URL="https://bugzilla.oracle.com/"

ORACLE_BUGZILLA_PRODUCT="Oracle Linux 7"
ORACLE_BUGZILLA_PRODUCT_VERSION=7.5
ORACLE_SUPPORT_PRODUCT="Oracle Linux"
ORACLE_SUPPORT_PRODUCT_VERSION=7.5
```
I figured 'Oracle' would be enough to give it away